Update Rule “do-you-use-voice-recordings-when-appropriate/rule” (#10585)
SSWConsulting/SSW.Rules.Content: rules/do-you-use-voice-recordings-when-appropriate/rule.md

---
type: rule
title: Communication - Do you use videos or voice recordings instead of sending
  long emails?

> seoDescription: Do you use videos or voice recordings instead of sending long emails?

Reviewer: Hi @Pam623 

- [ ] 1. Can you get a better SEO description for this (it's currently the title repeated)

**Tip:** USe ChatGPT

Author: Done.

uri: do-you-use-voice-recordings-when-appropriate
authors:
  - title: Adam Cogan
    url: https://ssw.com.au/people/adam-cogan
related: []
redirects:
  - communication-do-you-use-videos-or-voice-recordings-instead-of-sending-long-emails
created: 2014-02-25T15:46:04.000Z
archivedreason: null
guid: 5c824e64-1491-4957-baa2-3e18bdd00299
---

> Sometimes it is quicker to explain something verbally than it is to write a comprehensive email. This is particularly true when explaining an idea or concept relating to a product, and you need to go backwards and forwards many times to clarify details. If you can't easily speak directly with someone, due to schedule, travel, or time zone constraints try sending a voice recording instead.

Reviewer: Hi @Pam623 

This paragraph is too long

- [ ] 1. Please break it in 2 (before _"If you can't easily"_)

Author: Done.

<!--endintro-->

Sending videos or voice recordings can be thought of as a slow paced conversation. There are many benefits to sending voice messages instead of email when iterating over concepts and ideas.

> **Advantages of Using Videos or Voice Recordings:**

Reviewer: Hi @Pam623 

Instead of bold text, these should be headings 2

- [ ] 1. Please update this and lines 31 and 39 to be headings

Author: Done.

> 1. Personal Touch: They convey emotion and emphasis more naturally, making communication feel more personal.

Reviewer: Hi @Pam623 

These lists would look better if the prefix is bolded, the separator is a dash instead of a colon, and [no full stops](https://www.ssw.com.au/rules/avoid-full-stops/)

from

> 1. Personal Touch: They convey emotion and emphasis more naturally, making communication feel more personal.

To:

> 1. **Personal Touch** - They convey emotion and emphasis more naturally, making communication feel more personal

- [ ] 1. Please apply to all list items in the rule

**Video Recording Apps:**

* [Loom](https://www.loom.com/): Easy-to-use tool for recording quick video messages and screen shares, ideal for tutorials and feedback.
* [Snagit](https://www.techsmith.com/snagit/): Offers powerful screen capture and recording capabilities with editing features for polished videos.
* [Zoom for Screen Recording](https://www.zoom.com/en/products/screen-recorder/features/screen-recording/): Widely used video conferencing tool that also allows for high-quality screen recording.
* QuickTime Player (Mac): Built-in option for Mac users to record simple video and audio content easily.
* [Lark](https://www.larksuite.com/en_sg/) (aka [FeiShu](https://www.feishu.cn/)): All-in-one collaboration platform with built-in recording features, ideal for documenting meetings or sharing project updates across teams.
* [Tencent Meeting (China)](https://meeting.tencent.com/): Combines stable conferencing with built-in recording, useful for sharing key takeaways.
* [DingTalk (China)](https://www.dingtalk.com/en): Workplace-focused tool for quick status reports or task updates.

> For videos, you should put them on youtube as unlisted videos and send the link

Reviewer: ```suggestion
For videos, you should upload them to YouTube (as unlisted) and share the link.
```
